docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+---
+title: Reordering relations
+description: Use the REST API to manage the order of relations
+canonicalUrl: https://docs.strapi.io/developer-docs/latest/developer-resources/database-apis-reference/rest/relations-reordering.html
+---
+
+<style lang="scss" scoped>
+@media (min-width: 1536px) {
+  .custom-block.api-call > .response {
+    flex: 0 0 32%;
+  }
+}
+</style>
+
+# Managing relations through the REST API <BetaBadge />
+
+Relations between content-types can be managed through the [admin panel](/user-docs/latest/content-manager/managing-relational-fields.md#managing-multiple-choices-relational-fields) or through [REST](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md) requests sent to the Content API.
+
+Relations can be added, updated, or removed through the Content API by passing parameters in the body of the request:
+
+|  Parameter name | Description | Type of update |
+|-----------------|------------------|----------------|
+| [`connect`](#connect)       | Adds relation(s).<br /><br />Can be used in combination with `disconnect`.<br /><br />Can be used with [positional arguments](#relations-reordering) to define an order for relations.    | Partial
+| [`disconnect`](#disconnect)    | Removes relation(s).<br /><br />Can be used in combination with `connect`. | Partial
+| [`set`](#set)           | Sets relations, replacing all the existing ones.<br /><br />Must be used alone.  | Full
+
+Using `connect` allows adding [positional arguments](#relations-reordering) to define an order for relations.
+
+<br/>
+
+## `connect`
+
+Using `connect` in the body of a request performs a partial update, adding specified relations.
+
+`connect` accepts either a shorthand or a longhand syntax.
+
+| Syntax type | Syntax example |
+| ------------|----------------|
+| shorthand   | `connect: [2, 4]`
+| longhand    | ```connect: [ {id: 2}, {id: 4} ]``` |
+
+You can also use the longhand syntax to [reorder relations](#relations-reordering).
+
+`connect` can be used in combination with [`disconnect`](#disconnect).
+
+<br />
+
+
+:::::: tabs card
+
+::::: tab Shorthand syntax example
+
+::: request Example request using the shorthand syntax
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: { 
+      connect: [2, 4],
+    }
+  }
+}
+```
+
+:::
+
+:::::
+
+::::: tab Longhand syntax example
+
+::: request Example request using the longhand syntax
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: { 
+      connect: [
+        { id: 2 },
+        { id: 4 }
+      ], 
+    }
+  }
+}
+```
+
+:::
+
+:::::
+::::::
+
+
+### Relations reordering
```

Comment:
I find it a bit confusing this is nested into `connect`. What do you think about explaining the positional syntax outside of the `connect` / `disconnect` methods, because it applies to both of them?